Tests/Integration/Model/CardTest.swift

```diff
@@ -50,9 +50,352 @@ class CardTest: UpholdTestCase {
+            "\"type\": \"transfer\"," +
+            "\"message\": \"foobar\"," +
+            "\"network\": \"qux\"," +
+            "\"status\": \"pending\"," +
+            "\"RefundedById\": \"foobiz\"," +
+            "\"createdAt\": \"2014-08-27T00:01:11.616Z\"," +
+            "\"denomination\": {" +
+                "\"amount\": \"0.1\"," +
+                "\"currency\": \"BTC\"," +
+                "\"pair\": \"BTCBTC\"," +
+                "\"rate\": \"1.00\"" +
+            "}," +
+            "\"origin\": {" +
+                "\"AccountId\": \"fiz\"," +
+                "\"CardId\": \"bar\"," +
+                "\"accountType\": \"biz\"," +
+                "\"amount\": \"0.1\"," +
+                "\"base\": \"0.1\"," +
+                "\"commission\": \"0.00\"," +
+                "\"currency\": \"BTC\"," +
+                "\"description\": \"Foo Bar\"," +
+                "\"fee\": \"0.00\"," +
+                "\"rate\": \"1.00\"," +
+                "\"type\": \"card\"," +
+                "\"username\": \"foobar\"" +
+            "}," +
+            "\"destination\": {" +
+                "\"AccountId\": \"fuz\"," +
+                "\"accountType\": \"buz\"," +
+                "\"amount\": \"0.1\"," +
+                "\"base\": \"0.1\"," +
+                "\"commission\": \"0.00\"," +
+                "\"currency\": \"BTC\"," +
+                "\"description\": \"foo@bar.com\"," +
+                "\"fee\": \"0.00\"," +
+                "\"rate\": \"1.00\"," +
+                "\"type\": \"email\"" +
+            "}," +
+            "\"params\": {" +
+                "\"currency\": \"BTC\"," +
+                "\"margin\": \"0.00\"," +
+                "\"pair\": \"BTCBTC\"," +
+                "\"rate\": \"1.00\"," +
+                "\"refunds\": \"fizbiz\"," +
+                "\"ttl\": 30000," +
+                "\"type\": \"invite\"" +
+            "}," +
+            "\"normalized\": [{" +
+                "\"amount\": \"123\"," +
+                "\"commission\": \"0.00\"," +
+                "\"currency\": \"BTC\"," +
+                "\"fee\": \"1.00\"," +
+                "\"rate\": \"2.00\"" +
+            "}]," +
+            "\"fees\": [{" +
+                "\"type\": \"deposit\"," +
+                "\"amount\": \"0.30\"," +
+                "\"target\": \"origin\"," +
+                "\"currency\": \"USD\"," +
+                "\"percentage\": \"2.75\"" +
+            "}]" +
+        "}"
+
+        card.adapter = MockRestAdapter(body: json)
+        let transactionDenominationRequest = TransactionDenominationRequest(amount: "foo", currency: "bar")
+        let transactionDepositRequest = TransactionDepositRequest(denomination: transactionDenominationRequest, origin: "foobiz")
+
+        card.createTransaction(transactionDepositRequest).then { (transaction: Transaction) -> () in
+            XCTAssertEqual(transaction.createdAt, "2014-08-27T00:01:11.616Z", "Failed: Wrong transaction createdAt.")
```

Comment:
Only assert the `id` since we only need to make sure that it is a transaction that we are receiving.